internal/worker/server.go

```diff
@@ -718,6 +718,14 @@ func (s *Server) RequeueOrFinishJob(token uuid.UUID, maxRetries uint64, result j
 		}
 	}
 
+	if requeued {
+		jobInfo, err := s.jobInfo(jobId, nil)
+		if err != nil {
+			return fmt.Errorf("error requeueing job: %w", err)
```

Comment:
more like `error getting job details for prometheus metrics`
should we skip prometheus metrics here and try to continue, rather than fail?
although this will then probably fail some lines below…

Reply:
I think it's very unlikely that this call will fail if the job exists.